evaluator/src/test/kotlin/PackageRuleTest.kt

```diff
@@ -172,5 +172,52 @@ class PackageRuleTest : WordSpec() {
                 }
             }
         }
+
+        "hasVulnerability()" should {
+            "return true if a severity of a vulnerability is higher than the threshold" {
+                val rule = createPackageRule(packageWithVulnerabilities)
+                val matcher = rule.hasVulnerability("8.9", "CVSS3") { o1, o2 ->
```

Comment:
Instead of `o1` and `o2` I'd prefer names that make more clear which variable is the value, and which one the threshold. Also in `examples/rules.kts` below.